project/apps/auth-service/src/domain/ports/auth.service.ts

```diff
@@ -121,4 +138,20 @@ export class AuthService {
       this.handleError('sign in', new BadRequestException(error.message));
     }
   }
+
+  /**
+   * Signs out the user.
+   *
+   * @returns A promise that resolves to void.
+   * @throws Will throw an error if the sign-out process fails.
+   */
+  async signOut(): Promise<void> {
+    try {
+      await this.authRepository.signOut();
+
+      this.logger.log('user signed out successfully');
```

Comment:
similar to the comment above, should log which user has signed out instead of just a generic message